lib/oper1.g

```diff
@@ -515,12 +519,27 @@ BIND_GLOBAL( "INSTALL_METHOD",
         # If the requirements do not match any of the declarations
         # then something is wrong or `InstallOtherMethod' should be used.
         if notmatch=0 then
-          Error("the number of arguments does not match a declaration of ",
-                NAME_FUNC(opr) );
+          if not GAPInfo.CommandLineOptions.N then          
+            Error("the number of arguments does not match a declaration of ",
+                  NAME_FUNC(opr) );
+          else
+            Print("IM: the number of arguments does not ",
```

Comment:
Why "IM: "?

Reply:
Perhaps "IM" is meant to be a shorthand for `InstallMethod`?

If one sees this message as a user, it's a bit unclear what it implies. Perhaps this would be better? "Warning, the number of ..."